packages/x-data-grid/src/components/toolbar/GridToolbarExport.tsx

```diff
@@ -32,7 +32,6 @@ export interface GridToolbarExportProps {
    * @default {}
    */
   slotProps?: { button?: Partial<ButtonProps>; tooltip?: Partial<TooltipProps> };
-  [key: string]: any;
```

Comment:
We should not be more explicit here, like `excelOptions:any` or some such; doing this will leak the type to community users as well which may not be good DX wise.

Reply:
however types are exported  for `excelOptions` from premium package https://github.com/mui/mui-x/blob/master/packages/x-data-grid-premium/src/hooks/features/export/gridExcelExportInterface.ts#L65-L6